src/utils/sformat.fs

```diff
@@ -862,7 +864,7 @@ namespace Microsoft.FSharp.Text.StructuredFormat
                                         let postText = m.Groups.["post"].Value // Everything after the closing bracket
```

Comment:
Up at line __832__:
we get x.GetType() again ... I suppose we can remove it and use the ty that we fetched in __827__
````
    let ty = x.GetType()
        if ty.IsGenericType && ty.GetGenericTypeDefinition() = typedefof<Lazy<_>> then
            Some (wordL (x.ToString()))
        else
            // Try the StructuredFormatDisplayAttribute extensibility attribute
            match x.GetType().GetCustomAttributes (typeof<StructuredFormatDisplayAttribute>, true) with
````